modules/grafana/templates/dashboards/publishing_api_overview.json.erb

```diff
@@ -829,6 +829,91 @@
         }
       ],
       "title": "New row"
+    },
+    {
+      "collapse": false,
+      "editable": true,
+      "height": "250px",
+      "panels": [
+        {
+          "aliasColors": {},
+          "bars": false,
+          "datasource": "Graphite",
+          "editable": true,
+          "error": false,
+          "fill": 1,
+          "grid": {
+            "threshold1": null,
```

Comment:
I would love to get the variables here [1] in to this template, but after far too long trying to work out how puppet works, I've had to give up, but if someone has any advice, please let me know :)

1: https://github.com/alphagov/govuk-puppet/blob/86350f0a6e2ebc9e4be9c1df96e24739b63d5ddf/modules/govuk/manifests/node/s_redis_base.pp#L17-L18